Source/ACE/Entity/WorldObject.cs

```diff
@@ -160,101 +160,101 @@ public virtual void SerializeCreateObject(BinaryWriter writer)
             if ((WeenieFlags & WeenieHeaderFlag.PuralName) != 0)
                 writer.WriteString16L(GameData.NamePlural);
 
-            if ((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0)
-                writer.Write(GameData.ItemCapacity);
+            if (((WeenieFlags & WeenieHeaderFlag.ItemCapacity) != 0) && GameData.ItemCapacity != null)
```

Comment:
Well I might be wrong - but first - the binary writer will not write a null.   The first part of that were we are checking the weenieHeaderFlag against the specific mask != 0 means the flag is set and we want to write out the value.   I just had to add the check to make sure we are in fact not trying to write a null.   I don't think that is wrong.   Can someone else verify how that should be handled?   I either need to add that check or assert that it is not null.   Please advise.